src/XliffTasks/build/XliffTasks.targets

```diff
@@ -128,7 +138,7 @@
   </Target>
 
   <Target Name="TranslateSourceFromXlf"
-          Condition="'$(EnableXlfLocalization)' == 'true'"
+          Condition="'$(EnableXlfLocalization)' == 'true' and '$(_IsExcludedBuild) != 'true'"
```

Comment:
Hang on, I don't think it's right to exclude this. We can opt to skip updating xlf safely, but if we skip *consuming* xlf, then we won't produce satellites. I think it is better to exclude only EnsureXlfIsUpToDate.

Reply:
I'm now wondering if we should bother with this change. I filed it because @kevinh-ms wondered about it when adding xliff tasks to testimpact. It seemed like a good idea, but it's proving harder to get right than I expected. I haven't heard any actual measurements where xliff tasks was seen as a drag on LUT or DT builds. In the case common case of no string changes, I believe the incrementality that we have is good enough to keep it from being an issue? Have you observed the slowness or has anyone reported it?

Reply:
Sounds very reasonable.  Then we can use the data to decide how worth it is to think about all these cases. :)

Reply:
Anecdotally, I've not noticed any significant slowness either (and I do dogfood Live Unit Testing on LiveUnitTesting.sln most days).  It would be interesting to collect a log (even a command line msbuild with performance summary), just to see what the actual cost is.  If it's < 20ms or so, I wouldn't sweat it if it's not a safe/straightforward fix.